.github/workflows/digit-ui.yaml

```
@@ -0,0 +1,45 @@
+  workflow_call:
+
+jobs:
+  docker_image-build:
+    if: (github.event_name == 'push' && (startsWith(github.ref, 'refs/heads/develop')))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Docker
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)